app/api/server/api.js

```diff
@@ -180,10 +210,24 @@ class APIClass extends Restivus {
 			});
 		};
 		routes
-			.map(nameRoute)
+			.map((route) => this.namedRoutes(route, endpoints, apiVersion))
 			.map(addRateLimitRuleToEveryRoute);
 	}
 
+	getFullRouteName(route, method, apiVersion = null) {
+		let prefix = `/${ this.apiPath || '' }`;
+		if (apiVersion) {
+			prefix += `${ apiVersion }/`;
+		}
+		return `${ prefix }${ route }${ method }`;
+	}
+
+	namedRoutes(route, endpoints, apiVersion) {
```

Comment:
this was `nameRoute` earlier on, I found this to be misleading returning an array